logfire-api/logfire_api/__init__.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+from contextlib import contextmanager
+import importlib
+import sys
+from typing import TYPE_CHECKING, Literal
+from unittest.mock import MagicMock
+
+try:
+    logfire_module = importlib.import_module('logfire')
+    sys.modules[__name__] = logfire_module
+
+except ImportError:
+    if not TYPE_CHECKING:  # pragma: no branch
+        LevelName = Literal['trace', 'debug', 'info', 'notice', 'warn', 'warning', 'error', 'fatal']
+        VERSION = '0.0.0'
+        METRICS_PREFERRED_TEMPORALITY = {}
+
+        def configure(*args, **kwargs): ...
+
+        class LogfireSpan:
```

Comment:
Please mimic this: https://github.com/pydantic/logfire/blob/770780d4e0f5610bd8cbd596029d0cb1e23f25ae/logfire/_internal/main.py#L1695-L1714